dev-python/h5py/h5py-2.3.1.ebuild

```diff
@@ -18,7 +18,7 @@ KEYWORDS="amd64 x86 ~amd64-linux ~x86-linux"
 IUSE="test examples mpi"
 
 RDEPEND="
-	sci-libs/hdf5:=[mpi=]
+	sci-libs/hdf5:=[mpi=,hl(+)]
```

Comment:
Hi, I don't think we need a revbump here, because the old h5py fails to build.

Besides, I have committed the fix in tree.  But not sure how to close this PR.